pkg/whiteblacklist/whiteblacklist_test.go

```diff
@@ -118,3 +118,24 @@ func TestExclude(t *testing.T) {
 		}
 	})
 }
+
+func TestStatus(t *testing.T) {
+	t.Run("status when whitelist", func(t *testing.T) {
+		item1 := "item1"
+		whitelist, _ := New(map[string]struct{}{item1: {}}, map[string]struct{}{})
+		actualStatusString := whitelist.Status()
+		expectedStatusString := "whitelisting the following items: " + item1
+		if actualStatusString != expectedStatusString {
+			t.Errorf("expected status %s but got %s", expectedStatusString, actualStatusString)
```

Comment:
`s/%s/%q`

make the status string quoted string will make the output more readable.

Reply:
BTW, make be you should also add a ut for more than one whitelist items.

Reply:
fixed and added more test for more than 1 items